WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsDetailsViewModel.swift

```diff
@@ -392,6 +418,10 @@ class SiteStatsDetailsViewModel: Observable {
         }
         ActionDispatcher.dispatch(PeriodAction.refreshPeriod(query: .postStats(postID: postID)))
     }
+
+    func refreshEmailsSummary() {
+        subscribersStore.updateEmailsSummary(quantity: 30, sortField: .opens)
```

Comment:
I've chosen the same quantity and sorting as on the web for now.

Reply:
nitpick: I think replicating the web here is safe, but I wonder why it was done this way. For example:
1. I see six emails on the Subscriber tab
2. I click "View more" to see the rest
3. The Emails screen shows emails, but it's not clear which are the ones I haven't seen on the previous screen

I'm not sure why the web would intentionally sort by post ID on one page and by opens on the next. The latter seems much more useful on both screens.